src/frontend/next/src/pages/_app.tsx

```diff
@@ -18,24 +20,44 @@ const App = ({ Component, pageProps }: AppProps) => {
     }
   }, []);
 
-  const [theme, setTheme] = useState(lightTheme);
+  const preferredTheme = usePreferredThemeColors();
+
+  const [theme, setTheme] = useState(preferredTheme);
```

Comment:
You shouldn't need to set this via state if you have it from the hook above.

Reply:
@humphd : How do I trigger rerender of the whole app if this was the case?